STYLE.md

```diff
@@ -30,7 +30,8 @@
   fiasco](https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use) for
   how to best handle this.
 * API-level comments should follow normal Doxygen conventions. Use `@param` to describe
-  parameters, `@return <return-type>` for return values.
+  parameters, `@return <return-type>` for return values. Internal comments for
```

Comment:
I don't have a strong opinion here. I guess I would say that in a perfect world, all methods, etc. would have doxygen comments. Realistically though, it's hard enough to get people doing it for public methods and class comments, and I think in most cases people will just not comment on private methods/members instead of going through the whole doxygen dance. So, giving into the reality of human nature I would tend to agree with @htuch. But happy to get some more opinions.